<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 73, 85);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);
background-color: rgb(80, 114, 123);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>691</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Scrapping With Pools</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="text">
     <string>Number of Processes:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="text">
     <string>Number of Threads:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_of_processes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="specialValueText">
     <string> </string>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_of_threads">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="specialValueText">
     <string> </string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>681</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Tahoma&quot;;
color: rgb(120, 160, 131);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>671</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Tahoma&quot;;
color: rgb(145, 194, 158)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
